main.py

```diff
@@ -81,6 +83,28 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+@app.get("/experiment/submit/")
+async def submit_experiment(file: str, args: str = "{}") -> int:
+    """Submit the given experiment file.
+    
+    Args:
+        file: The path of the experiment file.
+        args: The arguments to submit which must be JSONifiable to a dictionary.
+          Each key is an argument name and its value is the value of the argument.
+    
+    Returns:
+        The run identifier, an integer which is incremented at each experiment submission.
+    """
+    expid = {
+        "log_level": logging.WARNING,
+        "class_name": None,
```

Comment:
Is this only necessary when there are more than one experiment class in a single file?

Reply:
As I understand, it is correct!

In [tools.py](https://github.com/m-labs/artiq/blob/454597915af820c94f8828a9e6cbcba03a03e600/artiq/tools.py#L94), I expect the parameter is used to find one of several experiments.

Reply:
Thank you for the reference!